examples/gateway-grpcroute.yaml

```diff
@@ -82,3 +82,6 @@ spec:
   - backendRefs:
     - name: grpcbin
       port: 443
+    matches:
```

Comment:
Why do we want to specify a match in the example file? The matches field is optional and can be omitted; IMO we should keep the examples as simpler as possible.

Reply:
Holdover from when I thought that was related to the bug causing the reflection errors.